src/server.h

```diff
@@ -233,9 +233,10 @@ extern int configOOMScoreAdjValuesDefaults[CONFIG_OOM_COUNT];
 
 /* Key argument flags. Please check the command table defined in the server.c file
  * for more information about the meaning of every flag. */
-#define CMD_KEY_WRITE (1ULL<<0)
-#define CMD_KEY_READ (1ULL<<1)
-#define CMD_KEY_INCOMPLETE (1ULL<<2)   /* meaning that the keyspec might not point out to all keys it should cover */
+#define CMD_KEY_WRITE (1ULL<<0)        /* "write" flag */
+#define CMD_KEY_READ (1ULL<<1)         /* "read" flag */
+#define CMD_KEY_SHARD_CHANNEL (1ULL<<2) /* "shard_channel" flag */
+#define CMD_KEY_INCOMPLETE (1ULL<<3)   /* "incomplete" flag (meaning that the keyspec might not point out to all keys it should cover) */
```

Comment:
if we're changing all these lines anyway, let's improve the indentation of the comments.